Gemfile

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'activemodel'
+gem 'activesupport'
+gem 'aws-sdk-cloudformation'
+gem 'aws-sdk-ec2'
+gem 'azure_mgmt_compute'
+gem 'azure_mgmt_network'
```

Comment:
Doesn't look like this is used.

Reply:
https://github.com/alces-software/cloudware/commit/0f7f475deeded76450ff284d9a1d30a42c4e3f4e